modules/programs/zsh.nix

```diff
@@ -142,6 +142,16 @@ let
     options = {
       enable = mkEnableOption "oh-my-zsh";
 
+      package = mkOption {
+        type = types.package;
+        default = pkgs.oh-my-zsh;
+        defaultText = literalExpression "pkgs.oh-my-zsh";
+        example = literalExpression "pkgs.oh-my-zsh";
+        description = ''
+          The oh-my-zsh package to install.
+        '';
+      };
```

Comment:
```suggestion
      package = mkPackageOption pkgs "oh-my-zsh" { };
```

Reply:
Oh interesting, I took reference from [`zoxide`](https://github.com/nix-community/home-manager/blob/36999b8d19eb6eebb41983ef017d7e0095316af2/modules/programs/zoxide.nix#L15-L22), I didn't know about this helper function. Thanks!